src/util/debounce.js

```diff
@@ -0,0 +1,10 @@
+export default function debounce(fn, ms) {
+  let timeout
+  // eslint-disable-next-line func-names
```

Comment:
I'm seeing a lot of `eslint-disable-next-line`, which makes me think perhaps the linting settings are too aggressive.

Reply:
yea, I was thinking the same.  I've created this issue to review our linting settings https://github.com/enjalot/blockbuilder-search/issues/73